packages/communication-react/stable-review/communication-react.api.md

```diff
@@ -646,6 +646,8 @@ export interface ChatMessage extends MessageCommon {
     messageType: 'chat';
     metadata?: Record<string, string>;
     // (undocumented)
+    metadata?: Record<string, string>;
```

Comment:
Duplicate prop, please remove this.
Looks like this got added during resolving merge conflicts.

Reply:
Done

Reply:
Looks like while merging you accidentally removed the  `metadata` doc block